packages/itwinui-react/src/core/Modal/Modal.test.tsx

```diff
@@ -240,3 +240,25 @@ it('should reset body overflow on closing and unmounting', () => {
   unmount();
   expect(document.body.style.overflow).not.toEqual('hidden');
 });
+
+it('should allow passing arbitrary props to internal elements', () => {
+  render(
+    <Modal
+      isOpen={true}
+      title='The title'
+      className='modal'
+      wrapperProps={{ className: 'wrapper' }}
+      backdropProps={{ className: 'backdrop' }}
+      titleBarProps={{ className: 'title-bar' }}
+    >
+      …
+    </Modal>,
+  );
+
+  const wrapper = document.querySelector('.wrapper') as HTMLElement;
```

Comment:
nit: Should we check if `wrapper` is truthy?

Reply:
I'm using `wrapper.querySelector` below, which would fail if it was not truthy.